js/packages/components/chat/message/UserMessageComponents.tsx

```diff
@@ -81,6 +82,7 @@ export const HyperlinkUserMessage: React.FC<{
 	const navigation = useNavigation()
 	const [{ margin, padding, column, border }, { scaleSize }] = useStyles()
 	const [isReadMore, setReadMore] = useState(false)
+	const { t }: any = useTranslation()
```

Comment:
you can use `t` like `<Text><>{t('whatever')}</></Text>` instead of casting the translation object to any